tools/gulptasks/dashboards/scripts-css.js

```diff
@@ -54,11 +54,17 @@ function dashboardsScriptsCSS() {
 
         const dataGridCSS = path.join('css', 'datagrid.css');
 
+        // TODO: datagrid should only be in a separate folder
         fslib.copyFile(
             dataGridCSS,
             path.join('code', 'datagrid', dataGridCSS)
         );
 
+        fslib.copyFile(
+            dataGridCSS,
+            path.join('code', 'dashboards', dataGridCSS)
+        );
```

Comment:
Sophie has already did it here: https://github.com/highcharts/highcharts/pull/19796